src/Symfony/Component/ExpressionLanguage/ExpressionLanguage.php

```diff
@@ -97,6 +97,19 @@ public function parse($expression, array $names)
         return $parsedExpression;
     }
 
+    /**
+     * @param Expression|string $expression The expression to check parse available
```

Comment:
`Validates the syntax of an expression.` as a title?

`The expression to validate` as a lable for the parameter